.github/workflows/push_pr.yml

```
@@ -0,0 +1,96 @@
+name: Push/PR pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
```

For other pipelines we use
```suggestion
on:
  push:
    tags:
      - v*
    branches:
      - main
  pull_request:
```
Might be a good opportunity to change it